tests/test_jx/test_deep_ops.py

```diff
@@ -953,7 +953,7 @@ def test_deep_agg_w_deeper_select_relative_name(self):
                 "select": {"value": "v.u", "aggregate": "sum"},  # TEST RELATIVE NAME IN select
                 "from": TEST_TABLE + ".a._b",
                 "edges": ["r.s"],  # TEST RELATIVE NAME IN edges
-                "where": {"not": {"eq": {"r.s": "b"}}}  # TEST RELATIVE NAME IN where
+                "where": {"ne": {"r.s": "b"}} # TEST RELATIVE NAME IN where
```

Comment:
do not change this test, we want to ensure the `ne` works.  You can copy the tests, so we test both `ne` and `not.eq`